Explorer/Assets/Scripts/Global/Dynamic/RealmLaunchSettings.cs

```diff
@@ -156,5 +161,28 @@ private bool IsRealmAWorld(string realmParam) =>
         private bool IsRealmAValidUrl(string realmParam) =>
             Uri.TryCreate(realmParam, UriKind.Absolute, out Uri? uriResult)
             && (uriResult.Scheme == Uri.UriSchemeHttp || uriResult.Scheme == Uri.UriSchemeHttps);
+
+        public void CheckStartParcelFeatureFlagOverride(IAppArgs appArgs, FeatureFlagsCache featureFlagsCache)
```

Comment:
Perhaps rename into `ApplyConfig` overload? As we could apply any other config from the feature flags (probably BDUF tho).

Reply:
Not so fond of changing it here, since this method includes logic very start parcel related (check if the parcel has changed in the Args)

So maybe we can keep this one separated